src/components/PopoverControl/LinkButton.js

```diff
@@ -12,7 +12,7 @@ const toggleLink = (editorState) => {
     return;
   }
   const href = window.prompt('Enter a URL');
-  const entityKey = Entity.create('LINK', 'MUTABLE', {href});
+  const entityKey = Entity.create('LINK', 'MUTABLE', {href, url: href});
```

Comment:
What parsers need the url param, it seems a bit redundant when it's already as the href?